src/OpenTelemetry.Api/Trace/SpanContext.cs

```diff
@@ -47,7 +47,7 @@ public SpanContext(in ActivityTraceId traceId, in ActivitySpanId spanId, Activit
         /// Initializes a new instance of the <see cref="SpanContext"/> struct with the given identifiers and options.
         /// </summary>
         /// <param name="activityContext">The activity context.</param>
-        internal SpanContext(in ActivityContext activityContext)
+        public SpanContext(in ActivityContext activityContext)
```

Comment:
Sometimes, we need to convert ActivityContext to SpanContext, so I had to change from internal to public.

Reply:
I think `ITextFormat` should use `ActivityContext` and not `SpanContext` since the `Activity` API is our preferred. So I would reverse what you have. In cases where Span shim layer needs to call this, instead of using the ctor we could add an implicit case operator?

Reply:
I see...let me try the reverse!

Reply:
Here's a bit more on the implicit cast. If you add this:
```csharp
    public readonly struct SpanContext
    {
        /// <summary>
        /// Converts a <see cref="SpanContext"/> into an <see cref="ActivityContext"/>.
        /// </summary>
        /// <param name="spanContext"><see cref="SpanContext"/> source.</param>
        public static implicit operator ActivityContext(SpanContext spanContext) => spanContext.ActivityContext;
    }
```
Then you can pass `SpanContext` anywhere `ActivityContext` is to be used.